src/Symfony/Component/Security/Http/RememberMe/RememberMeDetails.php

```diff
@@ -60,11 +66,6 @@ public function withValue(string $value): self
         return $details;
     }
 
-    public function getUserFqcn(): string
-    {
-        return $this->userFqcn;
-    }
```

Comment:
same here

Reply:
And I should implement new test by copying legacy one (existing one) ?

Reply:
Yes, exactly